api/all/src/main/java/io/opentelemetry/api/trace/SpanContext.java

```diff
@@ -65,33 +65,33 @@ static SpanContext createFromRemoteParent(
   }
 
   /**
-   * Returns the trace identifier associated with this {@code SpanContext}.
+   * Returns the trace identifier associated with this {@link SpanContext} as lowercase hex.
    *
-   * @return the trace identifier associated with this {@code SpanContext}.
+   * @return the trace identifier associated with this {@link SpanContext} as lowercase hex.
    */
-  String getTraceIdAsHexString();
+  String getTraceIdHex();
```

Comment:
Let's simplify even more :) `getTraceId()`. This means that we always refer to Trace ID, or Span ID, as the hex version by default. When we have a suffix like bytes, then it's the bytes version.

Reply:
Done. Will followup to update TraceId/SpanId classes to reflect this change.